LICENSE

```diff
@@ -1,6 +1,6 @@
-The MIT License (MIT)
+MIT License
 
-Copyright (c) Microsoft Corporation. All rights reserved.
+Copyright (c) 2016 Microsoft Azure
```

Comment:
Added 2016 looking at the git history.

Reply:
Adding the year makes sense to me, but I'm hesitant to change the name of the organization holding the copyright and removing the "All rights reserved." text. Being a fellow Microsofty, is there some official guidance you can send me offline on what this license file should hold?